data/events/2023/all-systems-go.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: All Systems Go
+date: '2023-09-13'
+displayDate: 'September 13 - 14, 2023'
+endsDate: '2023-09-14'
+location: 'Berlin, DE'
+link: 'https://all-systems-go.io'
+summary: 'The foundational user-space Linux conference'
```

Comment:
Do we want to plug @Xe's talk in the summary? I'm agnostic but it would follow the Gophercon example

Reply:
Good call